main/sdr.go

```diff
@@ -289,6 +300,92 @@ func (f *OGN) read() {
 	}
 }
 
+func (e *AIS) read() {
```

Comment:
Just to be sure: does this work correctly when the Stratux service crashes? Does rtl_ais exit if its pipe closes? Otherwise you'd potentially have to kill it off in the systemd service file.